pentasharp/Models/Entities/TaxiBookings.cs

```diff
@@ -11,6 +11,9 @@ public class TaxiBookings
         [Key]
         public int BookingId { get; set; }
 
+        [Required]
```

Comment:
remove data annotations as we agreed on removing them, add missing summaries for the whole model. rename this TaxiBooking model to the corresponding TaxiBookingRequest or TaxiBookingResponse and the folder shouldnt be named Entities

Reply:
@ganilala this is not fixed yet